### DISCUSSION TITLE
Feedback about the amazing bottom help bar

### DISCUSSION THREAD

**Discussion by zen0wu** [zellij-org/zellij#1102] [General]
Thanks for this amazing software, it's so good.

Two pieces of feedback:
1. Right now the bottom bar is two lines, on my screen, the right half of both lines are kind of wasted. I think maybe we could collapse these two lines: When we're in normal mode, it shows all the Ctrl+* shortcuts, and when one of the Ctrl+ shortcut is activated, the options are shown on the right half. (We could also optionally hide other shortcuts when one of them is activated)
2. Pressing 3 keys to switch pane seems a bit painful: Ctrl+P, arrow/hjkl, enter. Maybe there's a better way.

Apologize if any of these are missing the point, I just started using this and still learning.

**Comment by auronandace** [comment]
Just wanted to add my thoughts to this:

1. I prefer the way it is currently done. On smaller terminal sizes having the tips there ready without having to acitvate a mode feels more intuitive to me. There is an issue where you can suggest new tips: #938.
2. The tips show you can navigate panes via Alt+hjkl as long as the interface isn't locked with Ctrl+g.

The developers are very friendly and responsive here so they may follow up with further questions or suggestions when they have the time.
